openpype/lib/profiles_filtering.py

```diff
@@ -165,7 +165,8 @@ def filter_profiles(profiles_data, key_values, keys_order=None, logger=None):
             if match == -1:
                 profile_value = profile.get(key) or []
                 logger.debug(
-                    "\"{}\" not found in {}".format(key, profile_value)
+                    "\"{}\" not found in \"{}\": {}".format(value, key,
```

Comment:
This will  crash...

Reply:
Sorry it will not crash but will log nonsense. It didn't make sense before but it does not make sense even now.

It should be something like `"Key \"{}\" not found in profile. Profile keys: {}".format(key, str(tuple(profile.keys())))`.

NOTE: If match is `-1` it should mean that value is empty (not set).